Prace_domowe/Praca_domowa6/Jakub_Foltyn/PD6.ipynb

```diff
@@ -0,0 +1 @@
pPSH8ZHokek0aZow00XyithEJK8+WppXFfUyU44xpck/2Q6wzkfip81dc9S6My3yzP5udnybEqHCDLumVS5+q39zk0GtuNlZUVra2tLbxbcczBHp99glP8p8J1uH+rcKCpcuj5fuPIdyFCKqUxQTvhkfRDQmV/TyIcOsMLGa+PIUu8V6ViaY8uy/cjD1HPMAXGxbj9/ZD9ViZQBFWSQK6d4aH+jpohWLTlt/UwCK5Rhdk7esfUaDQajUaj0Wg0Go3GLseWmAMAy4hbkbHQYGlJq5FbZ7G0pOUVSwhepupYeo4yQaFfG4tLZcFlXFwrE3y5ZSyTlmWyMbfIpYWZ8blFP1kPWSKsKjlIW7xsoEq0leNizfhtfJ6ZMDFLgznyN1hmTT9y5Egn32nsGFZXV7V3796hhOEjjzwyHLvnnnskSR/4wAckzSfp41yA/KS3ClRlrjIRFZ+V5/zQoUOSpO9+97uSxvKfJEuVRsYA8om1d5ls51wqL2CWLkS23WJMf3j/KaMKc8B1MYwM1pd+L7roooU5TbE2WE9P0oP3kHPymeG6k98ShtqXvvQlSdKDDz44N0Zp9Fa+8MILrYsaO4a1tbU5OXGkV2qq3Jdj6liVZBBZ4RiyUiWbmkoYVjEd0D3J9EFXSdLDDz8saZQzdAZefU9I6P+faoPeoyQY+0z2S67HGR97p9zzuO5MxgRgjZy9wPqlXgX+21RJ36q/pXHPWO1XG43twmw20+bmZin/U8n/ePa7d5l9QMU6luZLBqcM5vO7KrmXjBzOcbYRSfRIUHrHHXdIkg4cODB3rrTo4U/Pv+sD5B3vPbqN60ijzuFYyrTre/ZBTzzxxFwbruPrDjNiipHkv0HuPTPJoOuiTERLPxWjrHonrdDMgUaj0Wg0Go1Go9FoNHY5tswcWF1dHSwPbrnAejxVGtDLfHFexqq4NSrbcgwLEOd6LBuWlbRy0YdbSugvY96w6PhY0sqTFmO3/mDtyZKIxOdJo+UnY14qpkOWBMt4tqpsEsjxuiUqy7JN9V/1m17KyjLYOQcaO43ZbDbog/e85z3D948//rikMb4fb/Jll10mad6rlvd/Wrbdc8T9nowB7vVHH310aHv33XdLGmXn5ptvnrvOwYMHh7Z43jMuN2Xfx8AxvGv87XPLEot464nlk0ZdhG5I793FF188tEWn4fXDCo6FH0+iJF1yySWSfl5GUFpkaLku4nni85TGmD1KA0ljfobHHntM0vh78fs7O8q9qMs8tY3Gm8Xa2lq5HwDptTtZGamqnyoeNZ/By+J8k4VYsTrzuc010Rn333//cOxb3/qWpNHbd+WVV0oay8q6zOGJzPH5OGEloL9hIqDbnDmQ+7dkQrrentozVW2TMbFsDwWyzGO1N6ty2DQa243ZbKYTJ04sfd6l7OW7lTTuK5LlzX1MjL9/lyWOq7KqyZyBxViVCMQTD1MA1iVj8zHkfi3fb1zGs2162atj7CWqEtccIwcTe6gs/S6N+5Onn35a0uL6etuTMQf895rKg8fnsve5KTRzoNFoNBqNRqPRaDQajV2OLTMHNjc3FzK6SqN3BysPVo4qPj0t11hI3Go0DDCsJ+k5w5vl10zLFVYTZwlUWXKl0Yrm162sxd6vAwsQn8S1uGUHi1cyCCqvO9byZCssy6cAprITS7W1zNtU3tXsLy3m3vb1119v9kBjx7C6uqozzzxzkCHiviTpfe97n6TROvsf//Efkkb98Pu///tDW85DBsk9kPpLGr1feLTxmOE5d681nmzki3Fyztlnnz03Fz8fD3rlXaRN6q+skuL9IJ9VnC+6KGNrWQdfV88pII2ePSz8fm28gOQnYL6wN6q8Mpn1l/G7lZ65wILgnCorMVbzU/HSNhpvBhsbG6VXKdkE+SyuvDhVLiM/l+t5m4wN9mdyevCSuelIzz79oOOefPLJoS1MKRhE5EPiHJc79An9s6dynUnlGXQH8oveIc+INMp/xX7w732eU1UbKkZGMgYqHTJVbSsrPEyNq9HYbsxmMx09erRkuiQbhvsz2ZKO9OwD1y88gzOWPfPDSaO8c818bnu/9AMLkbFUFdvSm54ef9d1jIF9HOd6lTz2ejC+2SfRxvUWjM9knzLuqqpAvn+xRlWVPD6X/QYJjlU5I/J3n0IzBxqNRqPRaDQajUaj0djl2BJzYDab6fjx4wtZpaUxKzceM+ItKisF1hMsQXjHkxXANf38tLxWWRiTkcDfbj3KjI94spZZ0/EuYjXi2m7BoS0WLOYCg0Cqs3D7uf59Wrf5e4rNUGEqq673l94K91oss8p7H1LtcW00thurq6vav3//oEP8fkOGr7vuOknSrbfeKkl66KGHJEn33Xff0JY42UsvvVTS6LWCdeB6C7lEtvGGE5fvVQCwJqdOgunkVUfSyo2Hv8poS3944DPPiuui9LxnZQJpZCCgTzJWr8qIS61ePHnMxXMukPkXr1/265mBGQ86mPXlGcJ6+Biyfi9r59Z1jwHuWN/GToE432VZ6vO5mkyC7M9R7aEyhh0Zr2J38xrJDqy8VVk5Jfc8fj564JprrpFU51BhnJm136+NHkEXZ2Umn1N6xqrKC2CKXQB8TsnWSA9sVa0gf9OqMkG1t2s0thuz2UwbGxtl/pOTVdWo3tEqRl4i4+bznc3lLlmSmb/Oc73BQkee2F9kricfZ46lQr7rMRbPCwerAB3GMa7puo21YT/Inop1cJZnsixYI2cigGSiTbHFTuWcSnc2c6DRaDQajUaj0Wg0Go3GUrRxoNFoNBqNRqPRaDQajV2OLScklOoSWySlgd7hCbekeboI1KpM6EKJioqGkgm3BhkclAAAIABJREFUoLs5TSupL9AmSIbjdFroF9BHoFqQRMLHyzGoH/SX1A0fT5Zc9DnRhv44xvx9XZM+y7WgqjidOpMcnUpCwCm6rc8pKUrLSmEwztNOO63LhzV2DCdOnNCPfvSjQaarMqVJv6IUDuEG0ijLlO5Dhp
```

Comment:
<p>W zadaniu chodziło o to, aby zdeformowane obrazki rzutować na przestrzeń komponentów wyznaczoną dla oryginalnych obrazków, następnie dokonać transformacji odwrotnej aby zobaczyć jak będą wyglądać odtworzone obrazki. Wówczas okazało się, że zrekonstruowane obrazki są dalekie od tych zdeformowanych. Konkluzją wówczas byłoby, że zaprezentowana metoda może służyć do detekcji outlierów.</p><p>W związku z tym w powyższej funkcji w pierwszej linijce ciała powinno być transform a nie fit_transform.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/352/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>